src/config.ts

```diff
@@ -19,10 +19,30 @@ export const CONFIG = {
     config: {
       name: 'Config',
       fields: {
-        projectId: {
+        vertexAiProjectId: {
           row: 1,
           col: 2,
         },
+        vertexAiLocation: {
+          row: 2,
+          col: 2,
+        },
+        vertexAiEndpoint: {
```

Comment:
We don't need this as mentioned earlier (sorry added comments in index.js instead of each individual file)